src/clinical-trial/gateways/file-repository/clinical-trial-file.repository.test.ts

```diff
@@ -46,7 +50,37 @@ describe('clinical trial file repository', () => {
       national_number: '2011-006209-83',
     }
 
+    const contactModel = new ContactModel(
+      new ContactDetailsModel(
+        'Institut Bergognié',
+        'Antoine',
+        'Italiano',
+        '5 avenue de l’opera',
+        'bordeaux',
+        'France',
+        '33076',
+        '01 23 45 67 89 ',
+        'aitaliona@bordeaux.fr',
```

Comment:
```suggestion
        'aitaliona@example.com',
```
C'est une bonne pratique de manière générale pour éviter le fait d'envoyer des emails.
https://datatracker.ietf.org/doc/html/rfc2606#section-3

A corriger sur les autres

Reply:
Fixed